File: monitoria-tmab-20191/listas1-2/lista1-3.cpp
#include <iostream>
#include <math.h>

using namespace std;

int main() {

    
int n, i;    
cout << "Qual a dimensão dos vetores? ";
cin >> n;

int x[n], y[n], w[n];
    
for(i = 0; i < n; i++){
    cout << "Digite a coordenada " << i << " do vetor x: ";
    cin >> x[i];
}

for(i = 0; i < n; i++){
    cout << "Digite a coordenada " << i << " do vetor y: ";
    cin >> y[i];
}

for(i = 0; i < n; i++){
    w[i] = x[i]+y[i];
}
    
cout << "x+y = [";
for(i = 0; i < n; i++){
    if (i != n-1){
        cout << w[i] << ",";
    } else {
        cout << w[i];
    }
}
cout << "]" << endl;

return 0;
}

File: monitoria-tmab-20191/listas1-2/lista1-3.py
import math

print('quantas coordenadas terão os vetores?')
n = int(input())

u = []
v = []
w = []

for i in range(n):
    print('Digite a coordenada '+str(i)+' do vetor u')
    u.append(float(input()))
for i in range(n):
    print('Digite a coordenada '+str(i)+' do vetor v')
    v.append(float(input()))
    
for i in range(n):
    w.append(u[i]+v[i])
    
print(w)